llvm/include/llvm/SYCLLowerIR/AddAspectNames.h

```diff
@@ -0,0 +1,32 @@
+//===--------------- AddAspectNames.h - AddAspectNames Pass ---------------===//
```

Comment:
I would rename it to something like `RecordSYCLAspectNames`. To me, record sounds better than add, but we should at least add `SYCL`, so it is clear that this is something SYCL-specific